app/assets/javascripts/angular/components/organizationSignatures/organizationSignatures.component.js

```diff
@@ -0,0 +1,88 @@
+import template from './organizationSignatures.html';
+import './organizationSignatures.scss';
+
+angular.module('missionhubApp').component('organizationSignatures', {
+    bindings: {
+        orgId: '<',
+    },
+    template: template,
+    controller: organizationSignaturesController,
+});
+
+function organizationSignaturesController(
+    httpProxy,
+    $scope,
+    $window,
+    envService,
+    $http,
+) {
+    this.signatures = [];
+    this.signatureData = [];
+    this.exportLink = '';
+    this.itemsPerPage = 10;
+
+    const getSignatures = (orgId, searchText) => {
+        const params = {
+            include: [],
+            ...(searchText ? { 'filters[q]': searchText } : {}),
+        };
+
+        return httpProxy.get(`/organizations/${orgId}/signatures`, params, {
+            errorMessage: 'error.messages.surveys.loadQuestions',
+        });
+    };
+
+    const prepareData = data => {
+        return data.map(data => {
+            return {
+                first_name: data.first_name,
+                last_name: data.last_name,
+                updated_at: data.updated_at,
+                code_of_conduct: data.code_of_conduct,
+                statement_of_faith: data.statement_of_faith,
+            };
+        });
+
+        return data;
+    };
+
+    const buildExportCsvLink = searchText => {
+        const params = {
+            access_token: $http.defaults.headers.common.Authorization.slice(7),
+            include: '',
+            format: 'csv',
+            ...(searchText ? { 'filters[q]': searchText } : {}),
+        };
+
+        const query = Object.keys(params).reduce((accumulator, key, index) => {
```

Comment:
```suggestion
        const query = Object.entries(params).reduce((accumulator, [key, value]) => {
```
works too. Then you don't have to manually index off of `params`.

Reply:
[`@babel/plugin-transform-runtime`](https://babeljs.io/docs/en/babel-plugin-transform-runtime) just adds babel helper code.

I've got it configured to use [@babel/preset-env](https://babeljs.io/docs/en/babel-preset-env) which should by default make it work with these browsers https://browserl.ist/. Idk if that's the best configuration but seems sane enough.